.github/ISSUE_TEMPLATE/question.md

```diff
@@ -0,0 +1,18 @@
+---
+name: Question
+about: Question relating to EdgeMesh.
+labels: kind/question
```

Comment:
For questions, now we can use https://github.com/kubeedge/kubeedge/discussions instead.

Reply:
fix